packages/qwik/src/core/v2/ssr/ssr-render-jsx.ts

```diff
@@ -228,6 +228,21 @@ export function toSsrAttrs(
       continue;
     }
 
+    if (isSignal(value)) {
+      const signalNode = ssrContainer.getLastNode();
+      // TODO(hack): last node is previous node, not current,
```

Comment:
Yeah, so this hack will not work as we may clobber the parent ID. So let's not do that.

So the issue is that we need to get a hold of the current element. But the current element has not been created, because we are in the process of creating the attributes for the current element. Hence, we have a circular dependency that we need to solve somehow. 🤔 

Here is my suggestion: 
1. In `toSsrAttrs` don't try to create a subscription, just put the signal directly into the `SsrAttrs`
2. Change the `SsrAttrs` so that it is `export type SsrAttrs = Array<string | Signal<any> | null>;`
3. in `writeAttrs`, check `isSignal(value)`, and if it is a signal, then do the code which you have here. Inside `writeAttrs` a call to `getLastNode()` will return a correct value. 

What do you think? I think that would be much cleaner.

Reply:
While this works there is a problem with this.
I need to import these things inside ssr-container and this is not allowed:

```
import { trackSignal } from '../core/use/use-core';
import { SubscriptionType } from '../core/state/common';
```